### ISSUE TITLE
Add to Analytics Page: List of Recent Editors

### ISSUE THREAD

**Issue by campgurus** [EBWiki/EBWiki#1099]
Editing articles is a core function of EBWiki that we want to encourage. Having a list of Editors active eg over the past 30 days will let us know how we are doing on that score.

Essentially, we can use the version.whodunnit method to got through all the versions created in the past 30 days and get a list of uniq users who have edited cases and then order them by number of cases edited during that period.

**Comment by campgurus** [comment]
screenshot of what this would look like:

<img width="1259" alt="screen shot 2017-11-13 at 3 43 35 pm" src="https://user-images.githubusercontent.com/1260659/32748239-a357bd0c-c889-11e7-8c3c-bc7fa793173a.png">

**Comment by rlgreen91** [comment]
Why would we restrict this info only to admins?  Additionally, what would be the point of this data?  Are we going to use it to highlight users?  Increase communications with people?

**Comment by campgurus** [comment]
I don't think we should restrict this analytics page at all. I think it should be publicly viewable.

I favor listing editors so we can see easily when new people pop up engaging with our content in that deeper way. I also think it could encourage them to see their name listed on that page, and maybe they do more editing than they otherwise would. These editors are viewable in the history tabs already, so this would just centralize the information for easier viewing.

**Comment by trystant** [comment]
@campgurus  @rlgreen91 So the content here is approved, and there's another discussion about making this information public.  Is this an accurate assessment?

**Comment by rlgreen91** [comment]
Is the content here approved?  My questions were more exploratory - I don't have a preference either way at the moment.  I still need to look at the PR. I might have more comments later, but they'll go either there or here.

**Comment by rlgreen91** [comment]
What is the difference between this and Issue #707?

**Comment by campgurus** [comment]
I still think this makes sense, both having the info available and making it public, if we ever get the analytics page working again.

**Comment by rlgreen91** [comment]
Does it, given that we're transitioning to Sumologic for our analytics?
